### PR TITLE
Update Zalo to 23.6.2

### PR DESCRIPTION
**Pull request by zeldrisho**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111661&drop=dogfoodAlpha

### CODE REVIEW

**Comment by giatrung2012** [comment]
@microsoft-github-policy-service agree

**Comment by giatrung2012** [comment]
https://zalo.me/download/zalo-pc?utm=90000 redirects to the Url used in the manifest.

File: manifests/v/VNGCorp/Zalo/23.6.2/VNGCorp.Zalo.locale.vi-VN.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: VNGCorp.Zalo
+PackageVersion: 23.6.2
+PackageLocale: vi-VN
+Publisher: VNG Corp.
+PackageName: Zalo
```
Comment (stephengillie) [inline manifests/v/VNGCorp/Zalo/23.6.2/VNGCorp.Zalo.locale.vi-VN.yaml:8]:
```suggestion
PackageUrl: https://zalo.me/pc
PackageName: Zalo
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**